app/javascript/src/Policies/index.jsx

```diff
@@ -4,6 +4,7 @@ import 'core-js/modules/es6.set'
 import 'core-js/modules/es6.map'
 import 'core-js/es7/array'
 import 'core-js/es7/object'
+import 'whatwg-fetch'
```

Comment:
I've tried before using another version of fetch and passing it as options to the `RSSA` action, but I reckoned it's better to use the already installed polifyll.